src/main/ts/ScriptLoader2.ts

```diff
@@ -0,0 +1,156 @@
+import { uuid } from './Utils';
```

Comment:
Why `ScriptLoader2`?

Reply:
Because the old one was `ScriptLoader` and I felt it was different enough to warrant a new name. ;-)